clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -69,6 +71,7 @@
 public class ConfigDef {
 
     public static final Object NO_DEFAULT_VALUE = new String("");
+    public static final String EXECUTABLE_PASSWORD_ENABLE_CONFIG = "executable.password.enable";
```

Comment:
The only use case someone would want to use executables as password is to be compliant of some security standard and for more security, leaving some of the passwords as plaintext, takes away both. I do not see any reason why someone would want to have some passwords as plaintext and some as executables. I plan on putting this in rejected alternatives in related KIP, which is yet to be put together.

That is an option that I intend to put in rejected section of KIP, which I still need to work on.